linera-service/tests/linera_net_tests.rs

```diff
@@ -708,6 +707,123 @@ async fn test_evm_call_wasm_end_to_end_counter(config: impl LineraNetConfig) ->
     Ok(())
 }
 
+#[cfg(with_revm)]
+#[cfg_attr(feature = "storage-service", test_case(LocalNetConfig::new_test(Database::Service, Network::Grpc) ; "storage_test_service_grpc"))]
+#[cfg_attr(feature = "scylladb", test_case(LocalNetConfig::new_test(Database::ScyllaDb, Network::Grpc) ; "scylladb_grpc"))]
+#[cfg_attr(feature = "dynamodb", test_case(LocalNetConfig::new_test(Database::DynamoDb, Network::Grpc) ; "aws_grpc"))]
+#[cfg_attr(feature = "kubernetes", test_case(SharedLocalKubernetesNetTestingConfig::new(Network::Grpc, BuildArg::Build) ; "kubernetes_grpc"))]
+#[cfg_attr(feature = "remote-net", test_case(RemoteNetTestingConfig::new(None) ; "remote_net_grpc"))]
+#[test_log::test(tokio::test)]
+async fn test_evm_execute_message_end_to_end_counter(config: impl LineraNetConfig) -> Result<()> {
+    use alloy_primitives::B256;
+    use alloy_sol_types::{sol, SolCall, SolValue};
+    use linera_base::vm::EvmQuery;
+    use linera_execution::test_utils::solidity::{get_evm_contract_path, read_evm_u64_entry};
+    use linera_sdk::abis::evm::EvmAbi;
+    let _guard = INTEGRATION_TEST_GUARD.lock().await;
+    tracing::info!("Starting test {}", test_name!());
+
+    let (mut net, client1) = config.instantiate().await?;
+
+    let client2 = net.make_client().await;
+    client2.wallet_init(&[], FaucetOption::None).await?;
+
+    let chain1 = client1.load_wallet()?.default_chain().unwrap();
+    let chain2 = client1.open_and_assign(&client2, Amount::ONE).await?;
+
+    let original_value = 35;
+    let moved_value = 5;
+
+    // Creating the API of the contracts
+
+    sol! {
+        function move_value_to_chain(bytes32 chain_id, uint64 moved_value);
+        function get_value();
+    }
+
+    let constructor_argument = Vec::new();
+
+    let instantiation_argument: Vec<u8> = u64::abi_encode(&original_value);
+
+    // Publishing the EVM contract and instantiating it on the two chains.
```

Comment:
Applications are _instantiated_ only on one chain. (I.e. `instantiate` is called only on the creator chain, at least for Wasm apps.)

Reply:
Removed.